src/ceph-volume/ceph_volume/tests/api/test_lvm.py

```diff
@@ -928,6 +928,7 @@ def test_vg_name_sans_mapper(self):
 class TestIsLV(object):
 
     def test_is_not_an_lv(self, monkeypatch):
+        monkeypatch.setattr(api.process, 'call', lambda x, **kw: ('', '', 0))
```

Comment:
if you use the `fake_call` fixture, there is no need to write the monkeypatch line here (it does the same)

Reply:
oooh, unless the patching doesn't work unless you use `api.process`?

Reply:
hmm good point actually. I just copied that from another line in this test file. Wondering what its patching there anyway, as `from ceph_volume.api import lvm as api` and patching process doesn't make much sense, does it?

Did this test collection maybe never quite patch things correctly?

Reply:
I think that some fixtures came after this became cumbersome, so a full cleanup/migration never happened. I would say: try it out, and if it works, use the fixture. If it doesn't (possible!) then this implementation is fine